### DISCUSSION TITLE
BTT Eddy

### DISCUSSION THREAD

**Discussion by SGK-IoT** [jschuh/klipper-macros#258] [Q&A]
My printer has a BTT Eddy installed. I want to modify the leveling macro's gcode so that it scans the bed instead of performing z hops. BED_MESH_CALIBRATE SCAN_MODE=rapid METHOD=scan is the new gcode.

How can I add that?

**Comment by cyberjak2k** [comment]
I believe that's what I did and I added it to my printer config under one
of the existing macros I just would need to look I think I followed some of
the details in the guide to do it
